src/modules/visitor-space/components/MaterialRequestBlade/MaterialRequestBlade.tsx

```diff
@@ -153,6 +159,14 @@ const MaterialRequestBlade: FC<MaterialRequestBladeProps> = ({
 		});
 	};
 
+	const onNoTypeSelected = () => {
+		toastService.notify({
+			maxLines: 3,
+			title: tText('selecteer-een-type'),
+			description: tText('selecteer-een-type-voor-je-aanvraag'),
```

Comment:
use regular dutch text without dashes
this text will get automatically extracted by the translation extraction script in the admin-core-ui
do not add these translation keys yourself